bl/evaluation/evaluator.go

```diff
@@ -16,12 +17,14 @@ type CLIClient interface {
 type Evaluator struct {
 	cliClient CLIClient
 	osInfo    *OSInfo
+	ciContext *ciContext.CIContext
```

Comment:
I'm not sure we need `ciContext` in the evaluator(TBH, maybe I added the `osInfo` and I don't think it was a good idea either). 
The reason why is because we need both of them only for `CreateEvaluation` so there's no point to keep them in the evaluator context.